components/brave_shields/browser/ad_block_base_service.h

```diff
@@ -47,6 +47,9 @@ class AdBlockBaseService : public BaseBraveShieldsService {
                           bool* did_match_exception,
                           bool* did_match_important,
                           std::string* mock_data_url) override;
+  base::Optional<std::string> GetCspDirectives(const GURL& url,
```

Comment:
is it clang-formatted?

Reply:
thought it was, but I guess this slipped through somehow. Should be fixed now.